pilot/pkg/config/kube/crd/config.go

```diff
@@ -42,6 +48,7 @@ func (in *IstioKind) DeepCopyInto(out *IstioKind) {
 	out.TypeMeta = in.TypeMeta
 	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
 	out.Spec = in.Spec
+	out.Status = in.Status
```

Comment:
Just find that since the beginning, this function is wrong. It does not really deepcopy.